shell/src/main/java/org/apache/accumulo/shell/commands/FateCommand.java

```diff
@@ -53,6 +67,51 @@
 
   private static final String USER = "accumulo";
 
+  // this class serializes references to interfaces with the concrete class name
+  private static class InterfaceSerializer<T> implements JsonSerializer<T> {
+    @Override
+    public JsonElement serialize(T link, Type type, JsonSerializationContext context) {
+      JsonElement je = context.serialize(link, link.getClass());
+      JsonObject jo = new JsonObject();
+      jo.add(link.getClass().getName(), je);
+      return jo;
+    }
+  }
+
+  // the purpose of this class is to be serialized as JSon for display
+  private static class ByteArrayContainer {
+    @SuppressWarnings("unused")
```

Comment:
Would be better to delete these, rather than suppress warnings. If they are needed, then please add a comment explaining why they should be left in place with the warning suppression (here, and other occurrences below).